### PR TITLE
673 make all console views more consistent

### PR DESCRIPTION
**Pull request by marynaKhromova**
Changes proposed in this pull request:

- create constant for empty text
- make empty fields consistent 
- make same order of `+Add ..` and `Search` buttons
- implement `Search` and `Filter` buttons as icons (without text)
- change design of current filter counter (the design has been discussed with our designers)

### CODE REVIEW

File: core/src/app/content/namespaces/operation/services/service-details/service-details.component.ts [hunk 1]
```diff
@@ -21,6 +22,7 @@ export class ServiceDetailsComponent implements OnInit, OnDestroy {
   public serviceDetailsUrl: string;
   public currentNamespaceSubscription: Subscription;
   public isSystemNamespace: boolean;
+  private emptyText = EMPTY_TEXT;
```
[the remark this answers is not in the document]
Comment (parostatkiem) [reply core/src/app/content/namespaces/operation/services/service-details/service-details.component.ts]: I confirm there will be errors on CI (and by `npm run build` also).

File: core/src/app/shared/constants/constants.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const EMPTY_TEXT = '-';
```
Comment (parostatkiem) [inline core/src/app/shared/constants/constants.ts]: Wouldn't it be better to have this on root level? As @jesusreal mentioned, this also needs to be used in Lambdas. Furthermore, in the near future we are going to take over all Console components which means this file would have to be duplicated `n` times.

File: core/src/app/content/namespaces/configuration/apis/apis.component.html [hunk 3]
```diff
@@ -4,15 +4,15 @@
...
-                Create API
-            </button>
             <y-list-search [hidden]="!initialized || showEmptyPage" [filterState]="filterState"
                 (onFilterChanged)="onFilterChanged($event)">
             </y-list-search>
             <y-list-filter [hidden]="!initialized || showEmptyPage || hideFilter" [filterState]="filterState"
                 (onFilterChanged)="onFilterChanged($event)">
             </y-list-filter>
+            <button fd-button (click)="navigateToCreate()" [options]="'light'" [glyph]="'add'">
+                Create API
```
[the remark this answers is not in the document]
Comment (marynaKhromova) [reply core/src/app/content/namespaces/configuration/apis/apis.component.html]: Here the changes were about the order of the buttons. `+` button should be everywhere with a text according to the design.

File: core/src/app/shared/constants/constants.ts (hunk 2 above)
Comment (jesusreal) [reply core/src/app/shared/constants/constants.ts]: I hope I understand correctly what you mean. Lambda is a different application than core and if you add it to the root and add the file somehow to the build of each application, this is a design topic which need some discussion IMHO.

Comment (parostatkiem) [reply core/src/app/shared/constants/constants.ts]: hmm, you are right. I just thought about the need of duplicating it many times. However, only Core and Lambda are going to use it (which I forgot previously) so the file will be duplicated only once which is not a big deal 👌

**Pull request merged**